packages/Schedular/src/schedule.ts

```diff
@@ -1,8 +1,23 @@
 import 'source-map-support/register';
+import request from 'request-promise';
```

Comment:
Can we use node-fetch as request-promise is deprecated

Reply:
Updated